### PR TITLE
Rewrite and clean code, add references. #68

### PR DESCRIPTION
**Pull request by AleksandrShvartz**

### CODE REVIEW

File: website/questmaker/schema.sql [hunk 1]
```diff
@@ -1 +1,145 @@
--- File will contain creating tables requests
\ No newline at end of file
+-- PostgreSQL database questmaker initialization
+
+
+--CREATE DATABASE questmaker;
+--\connect questmaker;
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:5]: If think, it's better to delete comments in lines 4, 5

File: website/questmaker/schema.sql [hunk 2]
```diff
@@ -1 +1,145 @@
...
+DROP  TABLE IF EXISTS statuses CASCADE;
+CREATE TABLE statuses (
+    status_id SERIAL PRIMARY KEY,
+    status CHARACTER VARYING(100) UNIQUE NOT NULL
+);
+
+DROP TABLE IF EXISTS authors CASCADE;
+CREATE TABLE authors (
+    author_id SERIAL PRIMARY KEY,
+    email CHARACTER VARYING(100)UNIQUE NOT NULL,
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:16]: (100) UNIQUE (add space))

File: website/questmaker/schema.sql [hunk 3]
```diff
@@ -1 +1,145 @@
...
+    name CHARACTER VARYING(100)  NOT NULL,
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:17]: two spaces before NOT

File: website/questmaker/schema.sql [hunk 4]
```diff
@@ -1 +1,145 @@
...
+CREATE TABLE quest_files (
+    entry_id SERIAL PRIMARY KEY,
+    quest_id INTEGER NOT NULL REFERENCES quests (quest_id) ON DELETE CASCADE,
+    f_id INTEGER NOT NULL REFERENCES files (f_id) ON DELETE CASCADE
+);
+
+DROP TABLE IF EXISTS questions_types CASCADE;
+CREATE TABLE questions_types (
+    q_type_id SERIAL PRIMARY KEY,
+	q_type_name CHARACTER VARYING(100) UNIQUE NOT NULL
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:76]: Extra tab

File: website/questmaker/schema.sql [hunk 5]
```diff
@@ -1 +1,145 @@
...
+    hint_id INTEGER NOT NULL REFERENCES hints (hint_id) ON DELETE CASCADE,
+    f_id INTEGER NOT NULL REFERENCES files (f_id) ON DELETE CASCADE
+);
+
+DROP TABLE IF EXISTS answer_options CASCADE; 
+CREATE TABLE answer_options(
+    option_id SERIAL PRIMARY KEY,
+    question_id INTEGER NOT NULL REFERENCES questions (question_id) ON DELETE CASCADE,
+    option_text TEXT NOT NULL,
+	points REAL NOT NULL DEFAULT 0.0,
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:107]: Extra tab on 107, 108

File: website/questmaker/schema.sql [hunk 6]
```diff
@@ -1 +1,145 @@
...
+	next_question_id INTEGER REFERENCES questions (question_id)ON DELETE CASCADE
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:108]: Add space before ON

File: website/questmaker/schema.sql [hunk 7]
```diff
@@ -1 +1,145 @@
...
+);
+
+DROP TABLE IF EXISTS movements CASCADE; 
+CREATE TABLE movements (
+    movement_id SERIAL PRIMARY KEY,
+    place_id INTEGER REFERENCES places (place_id) ON DELETE CASCADE,
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:114]: NOT NULL

File: website/questmaker/schema.sql [hunk 8]
```diff
@@ -1 +1,145 @@
...
+    question_id INTEGER NOT NULL REFERENCES questions (question_id) ON DELETE CASCADE,
+	next_question_id INTEGER REFERENCES questions (question_id)ON DELETE CASCADE
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:116]: Remove extra tab and add space before ON

File: website/questmaker/schema.sql [hunk 9]
```diff
@@ -1 +1,145 @@
...
+DROP TABLE IF EXISTS ratings CASCADE;
+CREATE TABLE ratings (
+    rating_id SERIAL PRIMARY KEY,
+    quest_id INTEGER NOT NULL UNIQUE REFERENCES quests (quest_id) ON DELETE CASCADE,
+    one_star_amount INTEGER NOT NULL DEFAULT 0,
+	two_star_amount INTEGER NOT NULL DEFAULT 0,
+	three_star_amount INTEGER NOT NULL DEFAULT 0,
+	four_star_amount INTEGER NOT NULL DEFAULT 0,
+	five_star_amount INTEGER NOT NULL DEFAULT 0
+);
```
Comment (IlyaP01) [inline website/questmaker/schema.sql:138]: Remove extra tabs in 134-137 lines

**Review by IlyaP01 — CHANGES_REQUESTED** [review]

File: website/questmaker/schema.sql (hunk 7 above)
Comment (IlyaP01) [reply website/questmaker/schema.sql:114]: place_id NOT NULL

**Review by IlyaP01 — APPROVED** [review]

**Review by gchevykalov — APPROVED** [review]

**Pull request merged**